### PR TITLE
Update configuration.md

### PR DESCRIPTION
**Pull request by LiorKogan**

### CODE REVIEW

File: docs/commands/graph.config-set.md [hunk 1]
```diff
@@ -1,5 +1,7 @@
 Set the value of a RedisGraph configuration parameter.
 
+Values set using `GRAPH.CONFIG SET` do not persist after server restart.
```
Comment (swilly22) [inline docs/commands/graph.config-set.md:3]: Maybe
Values set using `GRAPH.CONFIG SET` are not persisted
?

File: docs/docs/configuration.md [hunk 2]
```diff
@@ -45,6 +45,8 @@ GRAPH.CONFIG GET OPT1
 GRAPH.CONFIG GET *
 ```
 
+Values set using `GRAPH.CONFIG SET` do not persist after server restart.
```
Comment (swilly22) [inline docs/docs/configuration.md:48]: Same comment as above, also consider moving this line to follow:
```
GRAPH.CONFIG SET OPT1 VAL1
```

**Review by swilly22 — CHANGES_REQUESTED** [review]

File: docs/commands/graph.config-set.md (hunk 1 above)
Comment (LiorKogan) [reply docs/commands/graph.config-set.md]: I'm not sure if you suggest changing 'do no persist' to 'are not persisted' (I just changed it)
or do you recommend removing the 'after server restart'. I try to be extra-clear. I think 'after server restart' may make it clearer for other readers what we mean by persisted here.

File: docs/docs/configuration.md (hunk 2 above)
Comment (LiorKogan) [reply docs/docs/configuration.md]: Same

File: docs/commands/graph.config-set.md (hunk 1 above)
Comment (swilly22) [reply docs/commands/graph.config-set.md]: Yeah, It's kinda hard to put in words,
basically what we want to say is runtime changes to graph's configuration via `GRAPH.CONFIG SET` will not be persisted.

Maybe we can get some help phrasing this from @nermiller ?

Comment (nermiller) [reply docs/commands/graph.config-set.md]: How's this: Values changed using GRAPH.CONFIG SET during runtime do not persist after server restart.

Comment (LiorKogan) [reply docs/commands/graph.config-set.md]: @nermiller Sounds good to me ;-)

**Pull request merged**